src/Bonuses.ts

```diff
@@ -1,7 +1,7 @@
 export class Bonuses {
-  private level1To10Trains: number = 0
-  private level11To59Trains: number = 0
-  private postCreationAbilityPoints: number = 0
+  private level1To10Trains = 0
```

Comment:
the linter now complains for trivial type inference, so the `: number` annotations are removed